hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestWatchForCommit.java

```diff
@@ -282,6 +288,9 @@ public void testWatchForCommitForRetryfailure() throws Exception {
         // RuntimeException
         Assert.assertFalse(HddsClientUtils
             .checkForException(e) instanceof TimeoutException);
+        // client should not attempt to watch with
+        // MAJORITY_COMMITTED replication level
+        Assert.assertFalse(e.getMessage().contains("Watch-MAJORITY_COMMITTED"));
```

Comment:
Nit: Is it possible to match exception class rather than exception message? Makes future maintenance easier.